src/decode.c

```diff
@@ -171,6 +171,18 @@ Packet *PacketGetFromQueueOrAlloc(void)
     return p;
 }
 
+inline int PacketCallocExtPkt(Packet *p, int datalen)
+{
+    if (! p->ext_pkt) {
+        p->ext_pkt = SCCalloc(datalen, sizeof(char));
```

Comment:
Don't know if it actually matters, but "The calloc() function allocates memory for an array of nmemb elements of size bytes each and returns a pointer to the allocated memory." At least convention is to call it as: SCCalloc(1,data_len);